routes/tasks.js

```diff
@@ -13,5 +14,6 @@ router.post("/", authenticate, authorizeRoles([APPOWNER, SUPERUSER]), createTask
 router.patch("/:id", authenticate, authorizeRoles([APPOWNER, SUPERUSER]), updateTask, tasks.updateTask);
 router.get("/:username", tasks.getUserTasks);
 router.patch("/self/:id", authenticate, updateSelfTask, tasks.updateTaskStatus);
+router.patch("/self/:id", authenticate, updateSelfTask, tasks.updateTaskStatus, assignTask);
```

Comment:
why is the route mentioned 2 times? line no 16 & 17.

Reply:
removed it thanks